src/foam/nanos/dig/DIG.js

```diff
@@ -40,10 +44,66 @@ foam.CLASS({
     'AuthenticatedNSpecDAO'
   ],
 
+  sections: [
+    {
+      name: 'details'
+    },
+    {
+      name: 'supportDetails'
+    },
+    {
+      name: '_defaultSection',
+      permissionRequired: true
+    }
+  ],
+
   properties: [
     {
       name: 'id',
-      label: 'Request Name'
+      label: 'Request Name',
+      section: 'details'
+    },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.User',
+      name: 'createdBy',
+      storageOptional: true,
```

Comment:
created* properties shouldn't be storageOptional, only lastModified* properties.